bitcoind-wallet/bin/cli.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node --experimental-modules
```

Comment:
oh! maybe this needed `--experimental-json-modules`

Reply:
Ah, yes. Let's do eeet.